mlir/python/mlir/dialects/transform/extras/__init__.py

```diff
@@ -42,6 +44,46 @@ def __init__(
         super().__init__(v)
         self.parent = parent
         self.children = children if children is not None else []
+        self._normalform = Normalform
+
+    @property
+    def normalform(self) -> Type["Normalform"]:
```

Comment:
Just curious - I see this is techinically `typing.Type[ForwardRef('Normalform')]` but is this different from just writing `"Normalform"`?